src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,46 @@
-import React from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { TodosContext } from '../TodosContext/TodosContext';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+
+type Props = {
```

Comment:
```suggestion
interface Props = {
```
Use interface prefer. 
Type has unique ability but if you don`t need them use interface. It is sampled in scaled and has more interesting features.